cmd/template/dbdriver/postgres.go

```diff
@@ -0,0 +1,50 @@
+package dbdriver
+
+type PostgresTemplate struct{}
+
+func (m PostgresTemplate) Service() []byte {
+	return []byte(`package database
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"log"
+	"time"
+
+	_ "github.com/lib/pq"
+)
+
+type Service interface {
+	Health() map[string]string
+}
+
+type service struct {
+	db *sql.DB
+}
+
+func New() *service {
+	connStr := "user=pqgotest dbname=pqgotest sslmode=verify-full"
```

Comment:
I am going to add a general comment that will impact all of the DB implementations:
- using as it, curling localhost:8080/health fails due to connection not existing.
I am curious - should we include .env variables so that a user can update them and actually have a connection working OOB?
Because right now, if we include this feature and it doesnt work I feel a user can have poor experience.
If we allow them to update the .env (somehow?) and use a tool like godotenv, we can populate it and allow a proper connection.
And maybe (just a suggestion)if there is no .env we can just skip having the health check route implemented?
What are your thoughts?

Reply:
In my opinion, having the environment env available for the user to set makes more sense than disabling the route for the health check. because disabling the route could confuse the user more than showing an error log indicating that the connection fails due to the credentials not being set correctly.

Reply:
Good point on not bloating this PR - I am going to add an issue to include the Docker feature (https://github.com/Melkeydev/go-blueprint/issues/65)

However - my current question is that if someone spins up the Repo as is and they try to query the routes, it will fail out. 
isn't this a bit of a concern for a feature we are adding?
BTW I was also thinking due to the size of this PR, this needs to be a separate Major release at this point.

Reply:
After adding the env file and granting users the option to specify their database credentials before accessing the database health check, it is essential for the user to include the credentials in their env file even after incorporating the docker starter files. There seems to be no way around this requirement, as users must edit the env file to ensure proper functionality of the database.

similar to other frameworks like Laravel, users are required to generate the env file and provide the correct credentials to ensure that the started API route is working properly

Reply:
Also, about your question, I was thinking that in addition to adding the env file whenever the user chooses a database driver, we might also include a starter migration file. This could serve as a way to prompt the user to add their database credentials to the env file. 
What are your thoughts on this? Is it worth to tackle it in this PR, or should we add it in a separate PR to expedite the current one and get it merged asap?
@Melkeydev